tests/plugins/emoji/manual/emojimatchlabel.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.14.0, feature, emoji, 2583
+@bender-ckeditor-plugins: wysiwygarea, toolbar, emoji
+@bender-ui: collapsed
+@bender-include: ../_helpers/tools.js
+
+1. Focus editor.
+1. Type `:smiling_face:`
```

Comment:
```suggestion
2. Type `:smiling_face:` in the editor's content.
```

Reply:
It's rather obvious if in the previous step you are asked to focus the editor.